libpod/container_internal_linux.go

```diff
@@ -1650,6 +1650,12 @@ rootless=%d
 		}
 	}
 
+	// This must come after secretMounts, since secretMounts creates the /run/secrets dir
```

Comment:
I think it's worth elaborating a bit further and explain how secrets and mounted into the container.

1. We get the data from the secrets manager.
2. We copy the data to a file in the container's static dir.
3. We mount the previously copied file(s) into `/run/secrets`